web/src/modals_util.ts

```diff
@@ -0,0 +1,33 @@
+import {$t} from "./i18n";
+
+export const custom_time_unit_values = {
+    minutes: {
+        name: "minutes",
+        description: $t({defaultMessage: "minutes"}),
```

Comment:
Not sure if this would be better expressed as a type? It seems repetitive as is currently is.

Reply:
I did not get this. How to express this as type?

We can may be simplify the object as `minutes: $t({defaultMessage: "minutes"})`, but I think the current one is more readable.